test/ig/IGExporter.SUSHI-GENERATED-FILES.test.ts

```diff
@@ -48,7 +48,7 @@ describe('IGExporter', () => {
       expect(content).toMatch(/\| _updatePublisher.bat .*\| generated \| \s* \|/);
       expect(content).toMatch(/\| _updatePublisher.sh .*\| generated \| \s* \|/);
       expect(content).toMatch(
-        /\| input[\/\\]ImplementationGuide-fhir\.us\.minimal\.json \s*\| generated \| .*[\/\\]customized-ig[\/\\]config\.yaml, \{all input resources and pages\} \s*\|/
+        /\| input[\/\\]ImplementationGuide-fhir\.us\.minimal\.json \s*\| generated \| .*[\/\\]customized-ig[\/\\]sushi-config\.yaml, \{all input resources and pages\} \s*\|/
```

Comment:
This might not match what the user really has (if they have `config.yaml`) -- but I don't think I care.  We were already in that boat if they had `config.yml` anyway.